src/daemon/rpmostreed-daemon.c

```diff
@@ -277,6 +287,94 @@ rpmostreed_daemon_initable_init (GInitable *initable,
   return TRUE;
 }
 
+/* Returns TRUE if config file exists and could be loaded or if config file doesn't exist.
+ * Returns FALSE if config file exists but could not be loaded. */
+static gboolean
+maybe_load_config_keyfile (GKeyFile **out_keyfile,
+                           GError   **error)
+{
+  g_autoptr(GError) local_error = NULL;
+
+  g_autoptr(GKeyFile) keyfile = g_key_file_new ();
+  if (g_key_file_load_from_file (keyfile, RPMOSTREED_CONF, 0, &local_error))
+    {
+      *out_keyfile = g_steal_pointer (&keyfile);
+      sd_journal_print (LOG_INFO, "Reading config file '%s'", RPMOSTREED_CONF);
+      return TRUE;
+    }
+
+  if (!g_error_matches (local_error, G_FILE_ERROR, G_FILE_ERROR_NOENT))
+    {
+      g_propagate_error (error, g_steal_pointer (&local_error));
+      return FALSE;
+    }
+
+  sd_journal_print (LOG_WARNING, "Missing config file '%s'; using compiled defaults",
```

Comment:
Though are we planning to ship a commented-out version?  I'd say we should.

Reply:
Can you clarify? This is done in the PR, right? (It's at `src/daemon/rpm-ostreed.conf`). Or did you mean something else?